rest-e2e-tests/package.json

```diff
@@ -18,7 +18,7 @@
     "mocha": "6.2.2",
     "node-fetch": "2.6.0",
     "sleep-promise": "8.0.1",
-    "streamr-client": "3.1.3",
+    "streamr-client": "3.2.0",
```

Comment:
Version `3.2.0` has known problems and has been deprecated from npm. Please use `3.1.3` for now